test/test_save_as_netcdf.py

```diff
@@ -30,7 +30,6 @@ def nc_has_variable(ds, var_name):
     if var_name in ds.variables:
         return True
     else:
-        print(var_name, " is not a variable in the Dataset")
```

Comment:
In a way that is testing the data and not `pyugrid`, right? Or maybe I am missing something.